nixos/modules/services/networking/babeld.nix

```diff
@@ -87,9 +87,38 @@ in
       description = "Babel routing daemon";
       after = [ "network.target" ];
       wantedBy = [ "multi-user.target" ];
-      serviceConfig.ExecStart = "${pkgs.babeld}/bin/babeld -c ${configFile}";
+      serviceConfig = {
+        ExecStart = "${pkgs.babeld}/bin/babeld -c ${configFile} -I /run/babeld/babeld.pid -S /var/lib/babeld/state";
+        DeviceAllow = [ "/dev/null" "/dev/urandom" ];
```

Comment:
Isn't `PrivateDevices` enough?

Reply:
Isn't this more narrow and thus an improvement?

Reply:
My concern here is that the user is still able to see those devices and we must keep whitelisting them? With `PrivateDevices` the process would just get a "standard" layout of /dev without us having to care about which devices they try to access as none of them should be able to cause harm.

Reply:
Fair enough, I'll drop DeviceAllow completely then.